data.py

```diff
@@ -37,7 +37,7 @@ def get(universe, addr, n):
     @staticmethod
     def set(universe, addr, val):
         if (universe >= len(DMX_Data._universes)): return
-        if (bpy.context.scene.dmx.universes[universe].input != 'BLENDERDMX'): return
+        if (not bpy.context.scene.dmx.universes[universe] or bpy.context.scene.dmx.universes[universe].input != 'BLENDERDMX'): return
```

Comment:
```suggestion
        if (not bpy.context.scene.dmx.universes[universe]): return
        if (bpy.context.scene.dmx.universes[universe].input != 'BLENDERDMX'): return
```
Keeping a single check for early-return `ifs` adheres better to the project code style.